content-security-policy/script-src/script-src-strict_dynamic_double_policy_honor_whitelist.sub.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>Whitelists in a separate policy are honored with `strict-dynamic` in the script-src directive.</title>
+    <script src='/resources/testharness.js' nonce='dummy'></script>
+    <script src='/resources/testharnessreport.js' nonce='dummy'></script>
+
+    <!-- CSP served:
+    1) Content-Security-Policy: script-src 'strict-dynamic' 'nonce-dummy'
+    2) Content-Security-Policy: script-src 'self' 'nonce-dummy'
+    -->
+</head>
+
+<body>
+    <h1>Whitelists in a separate policy are honored with `strict-dynamic` in the script-src directive.</h1>
+    <div id='log'></div>
+
+    <script nonce='dummy'>
+        async_test(function(t) {
+            var handler = function(e) {
+                document.removeEventListener('securitypolicyviolation', handler);
+                t.unreached_func('No CSP violation report has fired.');
```

Comment:
`assert_unreached`. `t.unreached_func()` will return a function. You'd need to write `(t.unreached_func())()` for that to work.